gitops/git/github/github.go

```diff
@@ -15,6 +15,7 @@ var (
 	repoOwner = flag.String("github_repo_owner", "", "the owner user/organization to use for github api requests")
 	repo = flag.String("github_repo", "", "the repo to use for github api requests")
 	pat = flag.String("github_access_token", os.Getenv("GITHUB_TOKEN"), "the access token to authenticate requests")
+	githubHost = flag.String("github_host", "", "The host name of the private enterprise github, e.g. git.corp.adobe.com")
```

Comment:
The argument is only used to create a github client via `github.NewEnterpriseClient`. To be more clear about the purpose of the parameter, I'd name the argument as `github_enterprise_host.

Reply:
Good suggestion. I will update this.